### PR TITLE
Update kitchen config

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -1,32 +1,9 @@
...
-      - recipe[apt]
-  - name: centos-6.6
-    attributes:
-      osl-php:
-       packages:
-         apc:
-             php-apc: false
-         moar_packages:
-           - php-fpm
-           - php-gd
```
Comment (rettigs) [inline .kitchen.yml]: Not sure what to do about all this.  This looks like a lot of stuff for a kitchen config.  I wonder if it should be moved to recipes.

Comment (armiller) [inline .kitchen.yml]: They are both the same so we could move them to the `provisioner` attributes

File: metadata.rb [hunk 2]
```diff
@@ -7,3 +7,6 @@
 version          '0.1.2'
 depends          'php'
 depends          'build-essential'
+
+supports         'centos', '~> 6.0'
+supports         'ubuntu', '12.04'
```
Comment (ramereth) [inline metadata.rb]: Let's please remove ubuntu and replace it with CentOS 7.

**Comment by fahlmant** [comment]
@ramereth updated

File: .kitchen.yml (hunk 1 above)
Comment (ramereth) [inline .kitchen.yml]: @fahlmant these attributes need to still be included so that the tests work properly.

**Comment by ramereth** [comment]
@fahlmant unless I'm blind, I don't see an opcache recipe. Mind removing that suite?

**Comment by fahlmant** [comment]
@ramereth removed

File: .kitchen.yml [hunk 3]
```diff
@@ -35,22 +19,12 @@ suites:
...
+      - recipe[osl-php::apc]
     attributes:
-      php:
-        opcache:
-          opcache.interned_strings_buffer: 20M
-          opcache.max_accelerated_files: 10000
-          opcache.memory_consumption: 200M
+      osl-php:
+        packages:
+          php-apc: true
```
Comment (ramereth) [inline .kitchen.yml]: I think this attribute should be removed as this package doesn't exist.

**Comment by fahlmant** [comment]
@ramereth Okay all kitchen suites pass.

**Comment by fahlmant** [comment]
!bump patch ~

**Pull request merged**